task/ecosystem-cert-preflight-checks/0.2/ecosystem-cert-preflight-checks.yaml

```diff
@@ -22,6 +22,10 @@ spec:
       type: string
       description: The type of artifact. Select from application, operatorbundle, or introspect.
       default: "introspect"
+    - name: platform
```

Comment:
@mshaposhnik , can you help review this from the multi-platform controller perspective? I don't think that we would be able to use the `build-platforms` parameter directly as this isn't guaranteed to be _just_ an architecture specification.

Does this parameter also work with multi-platform controller (I don't think that we want VMs provisioned here... the analysis can all run within the cluster pods).